portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,48 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    <head>
```

Comment:
It looks like some of your old files are part of this commit, so why don't we wait for you to merge the first PR and then we'll see if these get updated? I know you've made a lot of changes to these files already!

Reply:
Sounds great, I'll wait.